week03_model_free/homework.ipynb

```diff
@@ -758,6 +758,58 @@
     "Also, don't forget to optimize ```learning_rate```, ```discount``` and ```epsilon``` params of model, this may also help to solve this env."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "!apt-get update\n",
+    "!apt-get install -y python-tk # The \"Pacman\" environment's requirement"
```

Comment:
I'm using Arch, where the system package manager is `pacman` instead of `apt`. Therefore, this cell will fail on my machine. I would suggest moving it to a Markdown cell ("there is a nice visualization ..., under Ubuntu it depends on ..., etc").